src/tools/ScenarioMeasurement/Startup/PerfCollect.cs

```diff
@@ -119,9 +138,9 @@ public void AddKernelKeyword(KernelKeyword keyword)
         public enum KernelKeyword
         {
             Empty,
-            ProcessLifetime,
-            Thread,
-            ContextSwitch
+            LTTng_Kernel_ProcessLifetimeKeyword,
+            LTTng_Kernel_ThreadKeyword,
+            LTTng_Kernel_ContextSwitchKeyword
```

Comment:
Do we need to put `LTTng` in these names for some reason? If we can keep our view a little more generic that seems better to me.

Reply:
hmm these are keywords defined in the new perfcollect script and we are just tostring() these enums to pass them as arguments to the script. These keywords exist only in ```PerfCollect.cs``` and lttng is indispensable part of perfcollect script. In other words, the concreteness of the naming seems equivalent to the scope of the context where it's used...

Reply:
Got it. Knowing we just pass these through is a good enough reason. Presumably we won't move off of LTTng anytime soon.